### ISSUE TITLE
POKE BUDDY

### ISSUE THREAD

**Issue by mcmagic1** [finndev/PokeBuddy#729]
good day github. why i cant login into POKE BUDDY. i already type FREE-BETA many times re-install many times but still it say's LOGIN FAILED. can you help me ? i really wanna use poke buddy thanks

**Comment by WolfKetchum** [comment]
Your Wi-Fi may be the problem, hey when you open pokebuddy does it open a error and says "Looks like github is down, try again later"? It means your Wi-Fi connection is super slow.

**Comment by Thorben24** [comment]
Also turn off firewall and antivir software and try it again.

**Comment by mcmagic1** [comment]
no there is no github is down or anything. when i loging in by the FREE-BETA it say's login failed.

**Comment by mcmagic1** [comment]
i already turn off the  firewall and my eset nod anti virus but still i cant login.

**Comment by Thorben24** [comment]
Try 2 basic steps:
Restart your computer.
Restart your router/modem.

**Comment by mcmagic1** [comment]
ok i will try

**Comment by mcmagic1** [comment]
STILL i cant login

**Comment by Thorben24** [comment]
Are you the administrator of your computer? Do you use a proxy server / vpn server? It seems like your ip adress is blocked by github api or something like that.

**Comment by mcmagic1** [comment]
yeah this is my personal computer. how can i been blocked by github. this is my 1st time to use github

**Comment by Thorben24** [comment]
Not sure. I think the problem is somewhere in your network settings. Do you use windows 10? Is any proxy / dns server added to your IPV4 settings in windows?
